web/handlers.go

```diff
@@ -177,15 +178,52 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("X-Frame-Options", "SAMEORIGIN")
 
 		// Add unsafe-eval to the content security policy for faster source maps in development mode
-		devCSP := ""
+		devCSPMap := make(map[string]bool)
 		if model.BuildNumber == "dev" {
-			devCSP += " 'unsafe-eval'"
+			devCSPMap["unsafe-eval"] = true
 		}
 
 		// Add unsafe-inline to unlock extensions like React & Redux DevTools in Firefox
 		// see https://github.com/reduxjs/redux-devtools/issues/380
 		if model.BuildNumber == "dev" {
-			devCSP += " 'unsafe-inline'"
+			devCSPMap["unsafe-inline"] = true
+		}
+
+		// Add supported flags for debugging during development, even if not on a dev build.
+		for _, devFlagKeyValueStr := range strings.Split(*c.App.Config().ServiceSettings.DeveloperFlags, ",") {
+			devFlagKeyValueSplit := strings.SplitN(devFlagKeyValueStr, "=", 2)
```

Comment:
Nit: too big of a variable name. Might be good to reduce it to `kvSplit`. Always prefer short names rather than long: https://github.com/golang/go/wiki/CodeReviewComments#variable-names.